### ISSUE TITLE
folder structure

### ISSUE THREAD

**Issue by havarhal** [equinor/standalones2rc#14]
Is it possible to have a folder structure that combines the standardized FMU setup with the old RC model set up?

**Comment by asnyv** [comment]
I propose a folder structure that handles the master simulation according to standard fmu format (making it e.g. webviz and fmu-ensemble compatible out of the box), and the same will also be possible with the slaves.

I propose this structure for rc model:
```
eclipse
    include # (master include)
    model # (master model)
    slaves
        slave1
            eclipse
                include
                model
        slave2
            eclipse
                include
                model
```

**Comment by asnyv** [comment]
Closed by #22
